tests/test_integration_upload.py

```diff
@@ -36,16 +52,17 @@ def test_upload_file():
     print("File upload successful, sialink: " + sialink2)
 
     headers = responses.calls[0].request.headers
-    assert headers["Content-Type"].startswith("multipart/form-data;")
+    assert headers["Content-Type"]
     assert headers["User-Agent"] == "python-requests/2.24.0"
     assert "Authorization" not in headers
 
+    params = responses.calls[0].request.params
+    assert params["filename"] == "file1"
+
     body = responses.calls[0].request.body
-    assert str(body).find('Content-Disposition: form-data; name="file"; \
```

Comment:
q: Why was this removed?

Reply:
same reason as above: the content data no longer has multipart decorations